lib/identity_cors.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class IdentityCors
-  FEDERALIST_REGEX = %r{\Ahttps://federalist-[0-9a-f-]+\.pages\.cloud\.gov\z}
+  FEDERALIST_REGEX = %r{\Ahttps:\/\/federalist-[0-9a-f-]+\.sites\.pages\.cloud\.gov\z}
```

Comment:
The `%r{...}` syntax instead of `//` means we don't need to escape the forward slashes, looks like Rubular isn't ready for that syntax, but the resulting regex is the same:
```ruby
> %r{\Ahttps:\/\/federalist-[0-9a-f-]+\.sites\.pages\.cloud\.gov\z} # %r{} without forward slashes escaped
=> /\Ahttps:\/\/federalist-[0-9a-f-]+\.sites\.pages\.cloud\.gov\z/
> %r{\Ahttps://federalist-[0-9a-f-]+\.sites\.pages\.cloud\.gov\z} # %r{} with forward slashes escaped
=> /\Ahttps:\/\/federalist-[0-9a-f-]+\.sites\.pages\.cloud\.gov\z/
> /\Ahttps:\/\/federalist-[0-9a-f-]+\.sites\.pages\.cloud\.gov\z/ # checking // for reference
=> /\Ahttps:\/\/federalist-[0-9a-f-]+\.sites\.pages\.cloud\.gov\z/
```

Rubocop may get mad at this as is?

```suggestion
  FEDERALIST_REGEX = %r{\Ahttps://federalist-[0-9a-f-]+\.sites\.pages\.cloud\.gov\z}
```